src/Symfony/Component/Translation/Translator.php

```diff
@@ -132,6 +132,10 @@ public function addResource($format, $resource, $locale, $domain = null)
             $domain = 'messages';
         }
 
+        if (null === $locale) {
+            @trigger_error(sprintf('Passing "null" to the third argument of the %s method has been deprecated since Symfony 4.4 and will throw an error in 5.0.', __METHOD__), E_USER_DEPRECATED);
```

Comment:
Passing a "null" domain to the "%s" method [...]

Reply:
Hum, see other similar messages... At least this should be "%s()" and not only %s
I'll let you decide what's best.

Reply:
updated